src/clj/rems/api/schema.clj

```diff
@@ -78,6 +81,7 @@
    :resource/ext-id s/Str
    :catalogue-item/id s/Int
    :catalogue-item/title LocalizedString
+   :catalogue-item/infourl OptionalLocalizedString
```

Comment:
Is OptionalLocalizedString needed here? I checked that localization-for function works so that it only fills the map with keys that actually exist, so there should not be nil values there. Am I missing something?

Reply:
Hmm okay maybe it only needs to be optional in the admin api. I'll check.

Reply:
Yep, unneeded.